draft-ietf-masque-h3-datagram.md

```diff
@@ -194,36 +203,49 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
+## HTTP Datagrams using Capsules
+
+When HTTP/3 Datagrams are unavailable or undesirable, HTTP Datagrams can be sent
+using the Capsule Protocol, see {{datagram-capsule}}.
```

Comment:
As we can also choose to send datagrams using capsules in HTTP/3, we should better talk about QUIC datagrams here. 
```suggestion
When QUIC Datagrams are unavailable or undesirable, HTTP Datagrams can be sent
using the Capsule Protocol, see {{datagram-capsule}}.
```

Reply:
I'm open to changing this, but if we did it would be more consistent with the rest of the document to use QUIC DATAGRAM frame. So perhaps "If the use of QUIC DATAGRAM frames is not possible, or the use is undesirable, HTTP datagrams can be sent using the Capsule Protocol, see {{datagram-capsule}}"

Reply:
We define the concept of "HTTP/3 Datagrams" in this document, so I personally find that clearer.

Reply:
I guess we should think twice about the sentence… re-considering the possibility to convert Datagram Capsules into HTTP/3 Datagrams over QUIC DATAGRAM frames, I guess we should keep HTTP/3 Datagrams here but strike the "or the use is undesirable" part as we can not prevent intermediaries from generating them anyway.